django/db/migrations/autodetector.py

```diff
@@ -33,6 +33,7 @@ def __init__(self, from_state, to_state, questioner=None):
         self.from_state = from_state
         self.to_state = to_state
         self.questioner = questioner or MigrationQuestioner()
+        self.apps_not_initial = [app for app, model in from_state.models]
```

Comment:
May as well make this a set here, since it's used only for containment checks? No purpose in keeping dupe app names in it.

Also, I think the name `existing_apps` would be clearer (I didn't understand this name until I looked how it was used), but that's bikeshedding, so do with it what you will.